### PR TITLE
[Refactor] Add K8sContextSwitcher Component

### PR DESCRIPTION
**Pull request by YashKamboj**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by YashKamboj** [comment]
This PR is ready to merge

File: ui/components/CustomAvatar.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    //   position : 'absolute',
+    //   top : 0,
+    //   left : 0,
+    //   width : '100%',
+    //   height : '100%',
+    //   borderRadius : '50%',
+    //   animation : '$ripple 1.2s infinite ease-in-out',
+    //   border : '1px solid currentColor',
+    //   content : '""',
+    // }
```
Comment (Abhishek-kumar09) [inline ui/components/CustomAvatar.js:20]: not required? Let's remove it.

File: ui/components/CustomAvatar.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {Badge, Box} from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+
+const StyledBadge = styled(Badge)(({ theme }) => ({
+  backgroundColor : '#44b700',
+    color : '#44b700',
```
Comment (Abhishek-kumar09) [inline ui/components/CustomAvatar.js:8]: do we have color palette for this?

File: ui/components/CustomAvatar.js (hunk 1 above)
Comment (Abhishek-kumar09) [inline ui/components/CustomAvatar.js:20]:
```suggestion
```

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: ui/components/CustomAvatar.js (hunk 2 above)
Comment (YashKamboj) [reply ui/components/CustomAvatar.js]: I just realised that this color doesn't change anything . @Abhishek-kumar09  Other changes done. 
The dco is failing again and again.

Comment (Abhishek-kumar09) [reply ui/components/CustomAvatar.js]: No problem for DCO rn

Comment (Chadha93) [reply ui/components/CustomAvatar.js]: Getting you covered on DCO here.

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**